federation/cmd/federation-apiserver/app/server.go

```diff
@@ -279,6 +280,10 @@ func defaultResourceConfig() *serverstorage.ResourceConfig {
 		extensionsapiv1beta1.SchemeGroupVersion.WithResource("ingresses"),
 		extensionsapiv1beta1.SchemeGroupVersion.WithResource("replicasets"),
 	)
+	// All batch resources except these are disabled by default.
+	rc.EnableResources(
```

Comment:
@jianhuiz you need to enable the api resources on a default api server run.
U just need to enable the flag on the run for e2e test.
(u can pull this here -> https://github.com/kubernetes/kubernetes/pull/46358/commits/1d564ae710d11f8b6d8468b98ced46490a4086af)